test/e2e/setup/e2e_setup.sh

```diff
@@ -14,13 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if ! command -v kind >/dev/null 2>&1; then 
+if ! command -v kind >/dev/null 2>&1; then
     echo "This script will install kind (https://kind.sigs.k8s.io/) on your machine."
     curl -Lo ./kind-amd64 "https://kind.sigs.k8s.io/dl/v0.12.0/kind-$(uname)-amd64"
     chmod +x ./kind-amd64
     sudo mv ./kind-amd64 /usr/local/bin/kind
 fi
 
+if ! command -v step >/dev/null 2>&1; then
+    echo "This script will install step (https://smallstep.com/docs/step-cli/) on your machine."
+    curl -Lo ./step_0.26.2_amd64.tar.gz "https://dl.smallstep.com/gh-release/cli/gh-release-header/v0.26.2/step_$(uname | tr '[:upper:]' '[:lower:]')_0.26.2_amd64.tar.gz"
+    tar -xzvf step_0.26.2_amd64.tar.gz
+    chmod +x ./step_0.26.2/bin/step
+    sudo mv ./step_0.26.2/bin/step /usr/local/bin/step
+    rm -rf ./step_0.26.2_amd64.tar.gz ./step_0.26.2
```

Comment:
It is worth extracting a version variable to define 0.26.2.

Reply:
done.